Dockerfile.dapper

```diff
@@ -28,7 +27,11 @@ ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH_arm64=arm64 GOLAN
 RUN curl -sLf https://storage.googleapis.com/golang/go1.16.4.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local
 
 RUN if [ "${ARCH}" == "amd64" ]; then \
-    curl -sL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s v1.40.1; \
+        curl -sL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s v1.40.1; \
+    fi
+# fugly workaround for https://bugzilla.suse.com/show_bug.cgi?id=1183043
```

Comment:
Recommend removing `fugly` word, just `workaround for ` should be enough. :)

Reply:
Fixed in latest patchset.